tutorials/fiori-tools-mockserver-opa/fiori-tools-mockserver-opa.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Run a SAP Fiori Elements Application using Mockserver
+description: Using mockserver and advance features in running SAP Fiori Elements application.
+auto_validation: true
+time: 30
+tags: [ tutorial>beginner, software-product>sap-fiori-tools, software-product>sap-fiori-elements, software-product>sap-business-application-studio ]
+primary_tag: software-product>sap-fiori
+author_name: Regina Sheynblat
+author_profile: https://github.com/reginasheynblat
+---
+
+## Prerequisites
+-An OData V4 service and an application based on that service that was generated by **SAP Fiori tools**.
+-Starting point could be to run through the tutorial [Create a Travel App with SAP Fiori Elements Based on OData V4 RAP Service](fiori-tools-rap-prepare-service).
+
+
+## Details
+### You will learn
```

Comment:
Please move this section above "Prerequisites"

Reply:
Please disregard this one. Everything is in its correct place.

Reply:
done

Reply:
ok - I will take it back as it was